cmake/Modules/FindPythonExtra.cmake

```diff
@@ -17,211 +17,75 @@
 #
 # CMake module for providing extra information about the Python interpreter.
 #
-# Output variables:
+# Output Targets:
+# - PythonExtra::Interpreter - an executable target that will invoke a version of Python
+#     matching CMAKE_BUILD_TYPE on platforms where that matters
 #
+# Output variables:
 # - PythonExtra_FOUND: True if a Python executable was found
-# - PythonExtra_EXTENSION_SUFFIX: The suffix for a Python extension, according
-#    to PEP-3149: https://www.python.org/dev/peps/pep-3149/
-# - PythonExtra_EXTENSION_EXTENSION: The extension for a Python extension. On
-#    Linux and Mac OSX equals to ".so", on Windows to ".pyd"
-# - PythonExtra_INCLUDE_DIRS: The paths to the directories where the Python
-#    headers are installed.
-# - PythonExtra_LIBRARIES: The paths to the Python libraries.
-# - PYTHON_SOABI: The shared library file name tag according to PEP-3149:
-#    https://www.python.org/dev/peps/pep-3149/
 #
-# Conditional output variables
+# Advanced Output variables
+# - PYTHON_EXECUTABLE: a path to a python interpreter
 # - PYTHON_EXECUTABLE_DEBUG: If the CMAKE_BUILD_TYPE is Debug and WIN32 is true
```

Comment:
I'd recommend saying these are deprecated in favor of `PythonExtra::Interpreter`, as that more closely matches the style in the newer module `FindPython3`. The `PYTHON_EXECUTABLE` variable in particular [came from FindPythonInterp](https://cmake.org/cmake/help/latest/module/FindPythonInterp.html).